File: no264/win-lose.rb
#!/bin/ruby
input = gets.chomp.split(' ')
N = input[0]
K = input[1]

if N === K
  puts 'Drew'
elsif (N.to_i + 1) % 3 == K.to_i
  puts 'Won'
else
  puts 'Lost'
end
  

File: no264/win-lose.go
package main

import "fmt"

func main() {
  var N, K int
  
  fmt.Scan(&N, &K)
  
  if (N + 1) % 3 == K {
    fmt.Println("Won")
  } else if N == K {
    fmt.Println("Drew")
  } else {
    fmt.Println("Lost")
  }
}